lib/statsd/instrument.rb

```diff
@@ -336,31 +335,38 @@ def backend
     @backend ||= StatsD::Instrument::Environment.default_backend
   end
 
-  # Emits a duration metric.
+  # @!method measure(name, value = nil, sample_rate: nil, tags: nil, &block)
```

Comment:
```suggestion
  # @!method measure(name, sample_rate: nil, tags: nil, &block)
```

if it is given a block, then it shouldn't be given a value as well